priv/lang/alpaca.es_MX.po

```diff
@@ -0,0 +1,76 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: alpaca\n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 1.8.12\n"
+"Last-Translator: Eric Bailey <eric@ericb.me>\n"
+"Language: es_MX\n"
+
+msgid "builtin_type_arity_error %(num_expected) %(num_supplied)"
+msgstr ""
+"Número incorrecto de parámetros de tipo proporcionados\n"
+"para el tipo incorporado \"%(type)\".\n"
+"Se esperaba $(num_expected), pero se consiguió %(num_supplied)."
+
+msgid "duplicate_definition %(id)"
+msgstr "Definición duplicada de \"%(id)\"."
+
+msgid "duplicate_type_definition %(id)"
+msgstr "Tipo \"%(id)\" ya se ha definido."
+
+msgid "function_not_exported %(mod) %(name)"
+msgstr "No hay ninguna función \"%(fun)\" que se exporte desde el módulo \"%(mod)\"."
+
+msgid "incomplete_expression"
+msgstr "Final inesperado de la expresión."
+
+msgid "invalid_bin_qualifier %(qualifier)"
+msgstr ""
+"Calificador binario no válido \"%(qualifier)\".\n"
+"Los calificadores válidos son \"end\",\n"
+"\"sign\", \"size\", \"type\" y \"unit\"."
+
+msgid "invalid_bin_type %(type)"
+msgstr ""
+"Tipo de parte binaria no válido \"%(type).\"\n"
+"Los tipos válidos son \"binary\", \"float\", \"int\" y \"utf8\"."
+
+msgid "invalid_endianness %(endianness)"
+msgstr "Endianidad no válido \"%{endianness}\". Quiso decir \"big\", \"little\", o \"native\"?"
+
+msgid "invalid_fun_parameter"
+msgstr "Patrón inválido para el argumento de la función."
+
+msgid "invalid_top_level_construct"
+msgstr "Esta construcción no puede aparecer en el nivel superior."
+
+msgid "module_rename %(old) %(new)."
+msgstr "Renombración del nombre del módulo de \"%(old)\" a \"%(new)\"."
```

Comment:
`Renombre de módulo de $old a $new` is the correct way to say `Module rename from $old to $new`. Is this probably what we mean here?